kategory/src/test/kotlin/kategory/data/SequenceKWTest.kt

```diff
@@ -0,0 +1,21 @@
+package kategory.data
+
+import io.kotlintest.KTestJUnitRunner
+import kategory.*
+import org.junit.runner.RunWith
+
+@RunWith(KTestJUnitRunner::class)
+class SequenceKWTest : UnitSpec() {
+    val applicative = SequenceKW.applicative()
+
+    init {
+        val eq: Eq<HK<SequenceKWHK, Int>> = object : Eq<HK<SequenceKWHK, Int>> {
+            override fun eqv(a: HK<SequenceKWHK, Int>, b: HK<SequenceKWHK, Int>): Boolean =
+                    a.ev().sequence.toList() == b.ev().sequence.toList()
```

Comment:
It'd be great to move this to an extension function, so it can be called directly like in here.